Makefile

```diff
@@ -16,21 +16,21 @@ install: build
 	mv ${BINARY} ~/.terraform.d/plugins/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/${OS_ARCH}
 
 test:
-	# build stub API server for the provider under test to talk to
-	go build github.com/kentik/community_sdk_golang/apiv6/localhost_apiserver
+	# build test API server for the provider under test to talk to
+	go build -o test_api_server github.com/kentik/community_sdk_golang/apiv6/localhost_apiserver
```

Comment:
Why not use the `tools./run_test_api_server.sh` script here?

Reply:
Done